src/app/ModalRenderer/ModalRenderer.tsx

```diff
@@ -44,6 +52,18 @@ export const ModalRenderer: React.FC = (): ReactElement | null => {
                     {notificationContent()} notifications on your email from {selectedNode.name}?
                 </ConfirmationDialogContainer>
             );
+        case ModalType.DeleteNode:
+            return (
+                <ConfirmationDialogContainer
+                    title="Remove node"
+                    confirmationButtonLabel="remove"
+                    showButtons={true}
+                    onConfirmation={handleRemoveNode}
+                    onCancel={() => dispatch(removeConfirmationDialog())}
```

Comment:
This is always the case with this cancellation button, it's better than to be inside `ConfirmationDialogContainer` rather than dispatching it here from `ModalRenderer` always